endpoints/openrtb2/video_auction.go

```diff
@@ -500,9 +502,31 @@ func mergeData(videoRequest *openrtb_ext.BidRequestVideo, bidRequest *openrtb.Bi
 		bidRequest.TMax = videoRequest.TMax
 	}
 
+	if videoRequest.Regs != nil {
```

Comment:
I didn't realize you were going to add the OpenRTB Regs into the video model directly. I expected something more along the lines of the simplified user object. This is totally fine with me, but in that case you can just set the bidRequest.Regs directly to the videoRequest.Regs... provided nothing else in the OpenRTB transform is writing out Regs in which case this code is safer.

Reply:
Updated the change here to setting the bidRequest.Regs directly to the regs from the videoRequest. Also updated unit test to have GDPR fields.